test_scripts/API/VehicleData/WindowStatus/001_GetVD_success_flow_with_windowStatus_param.lua

```diff
@@ -0,0 +1,27 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0261-New-vehicle-data-WindowStatus.md
+-- Description: Check that SDL successfully processes GetVehicleData with new `windowStatus` param.
+-- In case:
+-- 1) App sends GetVehicleData request with windowStatus=true to the SDL and this request is allowed by Policies.
+-- 2) SDL transfers this request to HMI.
+-- 3) HMI sends GetVehicleData response with all params of structure `windowStatus`.
+-- SDL does:
+-- 1) send GetVehicleData response to mobile with all parameters in `windowStatus` structure.
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/API/VehicleData/WindowStatus/common')
+
+--[[ Scenario ]]
+common.Title("Preconditions")
+common.Step("Clean environment", common.preconditions)
+common.Step("Update local PT", common.updatePreloadedPT)
+common.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+common.Step("Register App", common.registerApp)
+common.Step("Activate App", common.activateApp)
+
+common.Title("Test")
+common.Step("App sends GetVehicleData for windowStatus", common.getVehicleData, { common.windowStatusData })
```

Comment:
```suggestion
common.Step("GetVehicleData for windowStatus", common.getVehicleData, { common.windowStatusData })
```
This case is about processing GetVehicleData not only about sending, please perform update though all scripts also for others RPCs